components/Home.js

```diff
@@ -1,15 +1,40 @@
-import {
-  View, Text, Card, Colors,
-} from 'react-native-ui-lib';
+import { StyleSheet, View, Text, Card, Colors, Button} from 'react-native';
+import { useState } from 'react';
+import Camera from './Camera'
 
 export default function Home() {
-  return (
-    <View>
-      {/* <Card flex center onPress={() => console.log('pressed')}>
+    const [camera, setCamera] = useState(false);
+
+    function toggleCamera(){
+        setCamera(!camera)
+    }
+
+    if(camera){
+        return <Camera toggleCamera={toggleCamera}/>
+    }
+
+    return (
+    <View style={styles.container}>
+        <View>
+                  {/* <Card flex center onPress={() => console.log('pressed')}>
             <Card.Image source={{uri: 'https://github.com/wix/react-native-ui-lib/blob/master/demo/src/assets/images/card-example.jpg'}}/>
             </Card> */}
-      <Text $textSuccess text30>Text goes here</Text>
+      {/* <Text $textSuccess text30>Text goes here</Text> */}
+            <Text>PeepeePoopooPeepee</Text>
+            <Button 
+            title="Use Camera"
+            onPress={() => toggleCamera()}
+            />
+        </View>
     </View>
+    )
+};
+
+const styles = StyleSheet.create({
```

Comment:
We may want to start creating style sheets in another directory as @e-mbrown stated #12